Dockerfile

```diff
@@ -3,7 +3,10 @@ FROM php:7.1-fpm-alpine
 RUN apk add --no-cache --virtual .persistent-deps \
 		git \
 		icu-libs \
-		zlib
+		zlib \
+		wget \
+		ca-certificates \
```

Comment:
@20uf works fine! 

One thing that i noticed is that the size of docker has increased, because ubuntu's version is much heavier than alpine's docker version.

I have the custom of always using alpine versions, precisely because of the size, why did you choose to use the ubuntu image?